packages/gestalt/src/Video.css

```diff
@@ -34,4 +34,6 @@
     rgb(0 0 0 / 0.5) 66%,
     rgb(0 0 0 / 0.7)
   );
+  /* Workaround for the bug in iOS browsers with rendering transformed SVGs over video */
```

Comment:
curious is it happens with a scale flip, instead of a rotate
.arrow {
  transform: scale(-1, 1);
  transform-origin: center;
}

Reply:
solution by Dave